### PR TITLE
Modified Config to have satellite

### LINKED ISSUE #260: Add satellite imagery layer to map

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+## Unreleased
+
+### Fixed
+- Added Satellite imagery layer to the map [#260](https://github.com/IN-CORE/incore-playbook/issues/260)
+
+## [0.5.7] - 2023-04-05
```
Comment (longshuicy) [inline CHANGELOG.md]: Could you check the duplicated changelogs for 0.5.7 ?

File: src/utils/map.js [hunk 2]
```diff
@@ -118,6 +118,16 @@ export const basicLayers = new GroupLayer({
 				crossOrigin: "anonymous"
 			})
 		}),
+		new TileLayer({
```
Comment (longshuicy) [inline src/utils/map.js:121]: do you think it might be worthwhile to add the same satelite image to step 3-3? 
image

File: CHANGELOG.md (hunk 1 above)
Comment (Vismayak) [reply CHANGELOG.md]: They are different issues, no?

Comment (longshuicy) [reply CHANGELOG.md]: sorry let me rephrase... im just wondering why there is 0.5.7 above 0.5.8. And if you scroll down you'll see the duplicate 0.5.7. Likely rebase will fix this?

image

**Review by longshuicy — APPROVED** [review]
